src/hooks/useCheckAccessToken.ts

```diff
@@ -0,0 +1,27 @@
+import { useEffect, useState } from 'react';
+
+import { useNavigate } from 'react-router-dom';
+
+import useAccessToken from './useAccessToken';
+
+export default function useCheckAccessToken(): boolean {
```

Comment:
p3: 훅이 하는 일은 accessToken을 가지고 유효한지, 아닌지를 가르고 있는 것 같은데요. 우리가 집중해야 하는 것은 accessToken을 확인해서 인증된 유저인지 보는 것 보다는 현재 유저가 인증된 유저인지만 확인하고 싶은 것 같아요. 외부에서는 어떤 방식으로 인증된 유저인지 확인하는 것에는 관심이 없으니까요. 따라서 네이밍이 조금 더 범용적으로, 무엇을 하고 싶은지에 맞춰서 작성이 되도 좋을 것 같아요.

```suggestion
export default function useAuth(): boolean {

```